### ISSUE TITLE
Heating Control

### ISSUE THREAD

**Issue by rajivtanna** [ppieczul/org.openhab.binding.loxone#11]
I have successfully implemented this binding to my Loxone install with Google Home but just realized the binding does not work for heating controls? Is this correct?

**Comment by ppieczul** [comment]
Yes this is not implemented. Could you describe the use cases that are
important to you? Maybe they can be easily added.
Thanks
Pawel

W dniu pt., 28.12.2018 o 12:59 Rajiv Tanna <notifications@github.com>
napisał(a):

**Comment by rajivtanna** [comment]
Sure.

1) ask google the temperature in a particular room and get a response

2) turn on the heating in a certain room

3) turn off the heating in a certain room.

It seems that the home graph API already has this functionality so there
just needs to be a way of making this available from the Loxone miniserver.

**Comment by rajivtanna** [comment]
@ppieczul Happy to help test this although I will probably find it hard to actually dev it - can happily give it a go if you can give me some direction.

**Comment by ppieczul** [comment]
Hi Rajiv
Finally I found some time to make progress on the binding. Can you please
tell me which exactly functional blocks from the Loxone Config do you want
to be covered?
Thanks
Pawel

pon., 31 gru 2018 o 13:47 Rajiv Tanna <notifications@github.com> napisał(a):
